### PR TITLE
Add preferredLanguage field to OtpUser

### PR DESCRIPTION
**Pull request by philip-cline**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Adding a preferredLanguage field to the OtpUser class to store language preferences introduced in this PR (through UI selection).

### CODE REVIEW

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks amazing! And unit tests are passing now via method of "wait and try again"

**Comment by leonardehrenfried** [comment]
> Looks amazing! And unit tests are passing now via method of "wait and try again"

I highly effective strategy!

File: src/main/java/org/opentripplanner/middleware/models/OtpUser.java [hunk 1]
```diff
@@ -41,6 +41,9 @@
      */
     public String phoneNumber;
 
+    /** The user's preferred language **/
+    public String preferredLanguage;
```
Comment (binh-dam-ibigroup) [inline src/main/java/org/opentripplanner/middleware/models/OtpUser.java]: Can you add an example in the comment?

**Review by binh-dam-ibigroup — APPROVED** [review]

File: src/main/java/org/opentripplanner/middleware/models/OtpUser.java (hunk 1 above)
Comment (philip-cline) [reply src/main/java/org/opentripplanner/middleware/models/OtpUser.java]: Updated in https://github.com/ibi-group/otp-middleware/pull/164/commits/4d1e10ecaf3ed18039d2a49a9863cb156d0cc9eb

Comment (binh-dam-ibigroup) [reply src/main/java/org/opentripplanner/middleware/models/OtpUser.java]: Haha I was kinda expecting that! Then it's probably best to rename this field to `preferredLocale`.

File: src/main/java/org/opentripplanner/middleware/models/OtpUser.java [hunk 2]
```diff
@@ -41,6 +41,12 @@
      */
     public String phoneNumber;
 
+    /**
+     *  The user's preferred language, in language tag format
+     *  e.g. 'en-US', 'fr-FR', 'es-ES', 'zh-CN', etc.
```
Comment (binh-dam-ibigroup) [inline src/main/java/org/opentripplanner/middleware/models/OtpUser.java]: Remove extra space between `*` and comment text.

File: src/main/java/org/opentripplanner/middleware/models/OtpUser.java [hunk 3]
```diff
@@ -41,6 +41,12 @@
...
+     * **/
```
Comment (binh-dam-ibigroup) [inline src/main/java/org/opentripplanner/middleware/models/OtpUser.java]: Remove extra star and space.

File: src/main/java/org/opentripplanner/middleware/models/OtpUser.java (hunk 1 above)
Comment (philip-cline) [reply src/main/java/org/opentripplanner/middleware/models/OtpUser.java]: Hmm it was a trap.. I'll update right now ☺️

File: src/main/java/org/opentripplanner/middleware/models/OtpUser.java (hunk 3 above)
Comment (philip-cline) [reply src/main/java/org/opentripplanner/middleware/models/OtpUser.java]: Updated in https://github.com/ibi-group/otp-middleware/pull/164/commits/1d06681bccdd0db9bfcd7de8d02454b7f0d5c15c

File: src/main/java/org/opentripplanner/middleware/models/OtpUser.java (hunk 2 above)
Comment (philip-cline) [reply src/main/java/org/opentripplanner/middleware/models/OtpUser.java]: Updated in https://github.com/ibi-group/otp-middleware/pull/164/commits/1d06681bccdd0db9bfcd7de8d02454b7f0d5c15c

**Comment by binh-dam-ibigroup** [comment]
Marking as blocked pending corresponding UI code approvals.

**Pull request merged**